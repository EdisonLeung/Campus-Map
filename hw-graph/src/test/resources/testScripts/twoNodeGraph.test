# Tests the creation of 2 nodes

# Create graph
CreateGraph twoNodes

# Add 2 nodes
AddNode twoNodes a
AddNode twoNodes b

# Print nodes
ListNodes twoNodes