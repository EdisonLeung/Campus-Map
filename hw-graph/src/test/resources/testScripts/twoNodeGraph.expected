# Tests the creation of 2 nodes

# Create graph
created graph twoNodes

# Add 2 nodes
added node a to twoNodes
added node b to twoNodes

# Print nodes
twoNodes contains: a b